### ISSUE TITLE
Problem with Neo4j server scripts Initial heap size set to a larger value than the maximum heap size

### ISSUE THREAD

**Issue by marco-brandizi** [neo4j/neo4j#13270]
### TL;DR

Try this:

```bash
export JAVA_TOOL_OPTIONS="-Xms1G -Xmx4G"
<neo-server>/bin/neo4j-admin
``` 

It fails, with the Java error: *Neo4j server scripts Initial heap size set to a larger value than the maximum heap size*

That's due to the fact the `neo4j-admin` enforces its own sizes for the JVM heap:

```bash
exec "$JAVACMD" -Xmx128m \
  -classpath "$CLASSPATH" \
 ...
```

If you try JAVA_TOOL_OPTIONS="-Xmx4G" instead, it works, although the Neo4j is ignoring the new limit. That might be problematic when the server scripts are invoked from another application (see below).

Furthermore, the `neo4j` script to start the server also has this 128m limit. Can't that become a serious performance issue, with complex queries or heavy workloads?

### My use case 

I'm downloading and running the Neo4j server from [this Maven plug-in](https://github.com/Rothamsted/neo4j-server-maven-plugin), which, in turn is triggered by a Maven build of [a complex project](https://github.com/Rothamsted/knetminer). The latter requires `JAVA_TOOL_OPTIONS=-Xms1G -Xmx4G`.

## Expected behavior

Can those limits be removed from the Neo scripts? Or, could you check JAVA_TOOL_OPTIONS to see if the invoker already set them?

## Actual behavior

See above.

**Comment by klaren** [comment]
Hello and thanks for reporting.

The neo4j command-line tools work by first starting up a "small" JVM instance to parse the config and generate the actual java command, and will then fork to another process with the configured heap size.

You can use the environment variable [JAVA_OPTS](https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/#_environment_variables), which is recognized by the neo4j command-line tool and forwarded to the forked JVM.

The problem with using `JAVA_TOOL_OPTIONS` to specify options, is that it would affect the small JVM instance that does the bootstrapping as well.
